Frontend/src/Login.jsx

```diff
@@ -41,7 +45,8 @@ export default function Login({apiKey, setApiKey, setUserData, setUserId, setCou
 
         
       } catch (e) {
-        console.log("login failed");
+        console.log(e.response.data.message);
+        setLoginResponse(e.response.data.message);
```

Comment:
this doesn't check if there's actually a response (a backend that is down or times out will cause the front end to simply stall on logging in forever)